### DISCUSSION TITLE
Sandboxie and Dota 2

### DISCUSSION THREAD

**Discussion by ZaksSaFita** [sandboxie-plus/Sandboxie#3789] [Q&A Feedback]
Hi guys,

Used to be able to run two instances of dota 2 with sandboxie a while ago. Now that i want to do it says launching and goes back to "play" state on steam. 
Is there a way to launch two dota 2 games now or did Valve made it not work?

Thanks in advance for help.
And sorry for bad English.

**Comment by xsmolasses** [comment]
3 launched as Application Compartment (NoSecurityIsolation=y) - "PLAY" didn't stick otherwise.

2 Sandboxes, 1 Steam account: all but one at REGION select: "Lost connection to Steam", which disables FIND MATCH.

2 Sandboxes, 2 Steam accounts: all at FIND MATCH give VAC message, and cannot proceed. You're on your own (AFAIC).
```
Valve Anti-Cheat was unable to verify
that your machine is secure. Failure to
fix this may result in the inability to
play online.
```
```
Steam Beta Branch:  Stable Client
Steam Version:  1709846872
Steam Client Build Date:  Wed, Mar 6 8:31 PM UTC -08:00
Steam Web Build Date:  Thu, Mar 7 9:17 PM UTC -08:00
Steam API Version:  SteamClient021
```

**Reply by ZaksSaFita** [reply]
So that means cant open two Dota 2 games to play arcade mod.

Update: moved back to version 1.12.9 and works normaly.

**Comment by ZaksSaFita** [comment]
Rollback to version 1.12.9 fixed it

**Reply by Nabyne1337** [reply]
won't they block you for this?

**Reply by joint13375-cell** [reply]
I can't download this version for Windows 11.
